server.js

```diff
@@ -30,8 +37,100 @@ app.set('views', viewsPath);
 app.set('port', process.env.PORT || appConfig.port);
 // Set the CookieParser middleware
 app.use(cookieParser());
+// Set Global publicKey
+app.set('nyplPublicKey', appConfig.publicKey);
+app.use(session({ secret: process.env.SESSION_SECRET || uuidv4() }));
 // Sets the server path to /dist
 app.use(express.static(distPath));
+// Use passport middleware for authentication
+app.use(passport.initialize());
+// use passport sessions
+app.use(passport.session());
+
+// Set up list of authorized users
+let authorized_users = undefined;
```

Comment:
As per our code review session, this should be a middleware function added to server/auth/

Using `req.authorizedUsers` to assign a global object to the express app instance.

Reply:
Thanks for this. I'm now making `user` a variable on the req object. As `authorizedUsers` is only seen in the module we shouldn't need to expose it, so having moved it as you suggested, I think we can love it as a module variable. You'll see in the updated PR.